### PR TITLE
Feature/update GitHub actions jdk

### PR DESCRIPTION
**Pull request by MaksimR-AVISPL**
Tests are skipped for now, until are refactored, considering all the recent changes. 
Just not to waste GitHub actions rolls on the failing tests + github actions jdk version update to the valid 14 instead of 9

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -9,13 +9,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - name: Set up JDK 9
+    - name: Set up JDK 14
       uses: actions/setup-java@v1
       with:
-        java-version: 9
+        java-version: 14
```
Comment (VladK-AVISPL) [inline .github/workflows/maven.yml:15]: Does it mean that compilation level is 14? We should use JDK 14, but compile level should be 8 for now (until we change that requirement) in order to be compatible with old CPXs

**Review by VladK-AVISPL — COMMENTED** [review]
Please see question inline.

File: .github/workflows/maven.yml (hunk 1 above)
Comment (MaksimR-AVISPL) [reply .github/workflows/maven.yml:15]: It's basically just a version of jdk that github is using for the build, but the compilation level is set in pom file - target version is 8. It is not really critical which version to have in here, but previously it's been 9 so I just figured it's better to get it aligned with either 8 or 14. So if you think this one is better to be kept as 8 - I will change it down to 8

Comment (VladK-AVISPL) [reply .github/workflows/maven.yml:15]: Understood. Leave it 14 then.

**Review by VladK-AVISPL — APPROVED** [review]

**Pull request merged**